server/api/routes/meeting.cjs

```diff
@@ -93,6 +94,81 @@ router.delete('/:meetingId', requireToken, isAdmin, async (req, res, next) => {
     next(err);
   }
 });
+router.post('/active/messages', requireToken, async (req, res, next) => {
+  try {
+    let correctRecip;
+    const meeting = await Meeting.findOne({
+      where: {
+        isClosed: false,
+        meetingStatus: 'confirmed',
+        [Op.or]: [{ userId: req.user.id }, { buddyId: req.user.id }],
+      },
+    });
+    if (meeting === undefined) {
+      res.status(404).send('User is not in a meeting that is confirmed');
+    }
+    console.log(typeof req.user.id);
+    if (req.user.id === meeting.userId || req.user.id === meeting.buddyId) {
+      if (req.user.id === meeting.userId) correctRecip = meeting.buddyId;
+      else correctRecip = meeting.userId;
+      const newMessage = await Message.create({
+        recipientId: correctRecip,
+        meetingId: meeting.id,
+        message: req.body.message,
+        senderId: req.user.id,
+      });
+      res.status(200).json(newMessage);
+    } else {
+      res.status(404).send(`User is not in meeting`);
+    }
+  } catch (err) {
+    next(err);
+  }
+});
+// to get the currently active meeting for said user
+router.get('/active/messages', requireToken, async (req, res, next) => {
+  try {
+    const meeting = await Meeting.findOne({
+      include: [
+        {
+          association: 'user',
+          attributes: ['firstName', 'lastName', 'fullName', 'avatarUrl'],
+        },
+        {
+          association: 'buddy',
+          attributes: ['firstName', 'lastName', 'fullName', 'avatarUrl'],
+        },
+        {
+          model: Message,
+        },
+      ],
+      where: {
+        isClosed: false,
+        meetingStatus: 'confirmed',
+        [Op.or]: [{ userId: req.user.id }, { buddyId: req.user.id }],
+      },
+    });
+    if (meeting) {
+      if (
+        req.user.id === meeting.userId ||
+        req.user.id === meeting.buddyId ||
+        req.user.role === 'admin'
+      ) {
+        res.json(meeting);
+      } else {
+        res.status(403).send('User is not found in meeting that is confirmed');
+      }
+    } else {
+      res
+        .status(404)
+        .send(
+          `Meeting not found with id ${req.params.meetingId} that's currently active/confirmed `
```

Comment:
You are not passing in any params so req.params.meetingId will always be undefined